Riot/Modules/Pills/PillTextAttachmentData.swift

```diff
@@ -108,6 +112,7 @@ struct PillTextAttachmentData: Codable {
         case isHighlighted
         case alpha
         case font
+        case maxWidth
```

Comment:
My only question is about encoding/decoding these pills - I'm not sure where/when we do it.

Does it make sense to encode a computed maxWidth with them? Could it be decoded to be displayed in a different size window (e.g. If the user is now using split view on an iPad)? If so, maybe it would make sense to not encode it, and instead set it back to `.greatestFiniteMagnitude`?